drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -903,7 +940,28 @@ function DragAndDropBlock(runtime, element, configuration) {
                 }
             }
         }
-    }
+    };
+
+    var sliderInitializer = function() {
```

Comment:
@murad-hubib The tests are breaking because this commit removes @jawad-khan 's `initializeSlider` method, and so the slider isn't getting created.

Can you please remove these commits and submit a separate PR for the styling changes, so we can get this functionality merged to the feature branch ASAP?